source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -62,12 +62,13 @@ export abstract class Dialog {
                 if (!this.preventLightDismissal) { this.close(); }
             };
 
-            let dialogFrameElement = document.createElement("dialog");
+            let dialogFrameElement = document.createElement("div");
             dialogFrameElement.className = "acd-dialog-frame";
             dialogFrameElement.style.width = this.width;
             dialogFrameElement.style.height = this.height;
             dialogFrameElement.style.justifyContent = "space-between";
-            dialogFrameElement.setAttribute("aria-modal", "true");
+			dialogFrameElement.setAttribute("aria-modal", "true");
```

Comment:
Indentation is off

Reply:
Sure, seems tabs/space issue. Will fix it